superset-frontend/src/dashboard/components/nativeFilters/FilterBar/state.ts

```diff
@@ -105,12 +105,9 @@ export const useInitialization = () => {
     }
 
     const numberOfLoadedCharts = Object.values(charts).filter(
-      ({ chartStatus }) => chartStatus !== 'loading',
+      ({ chartStatus }) => chartStatus === 'rendered',
```

Comment:
Just double checking: does this also correctly handle charts in error state?

Reply:
You are right.  I will revert it. sorry.